.pyup.yml

```diff
@@ -0,0 +1,25 @@
+# update schedule
+# default: empty
+# allowed: "every day", "every week", ..
+schedule: "every day"
```

Comment:
maybe once a week? :)

Reply:
I think what we're seeing now is the "problem" of not integrating upstream upgrades in due time, we can maybe talk about it at the dev meeting, but I think @pyup-bot should just go nuts and hit us with these updates every day so we don't linger. We can add deliberate filters as I'm doing in this PR for the things we have chosen not to update.